src/repr/src/row/encoding.rs

```diff
@@ -414,7 +419,11 @@ impl DatumToPersist for NullableProtoDatumToPersist {
             let col = col.downcast_ref::<Option<Vec<u8>>>()?;
             debug_assert!(validity.is_superset(col.logical_nulls().as_ref()));
             let (lower, upper, null_count) = proto_datum_min_max_nulls(col, ValidityRef::none());
-            let stats = BytesStats::Atomic(AtomicBytesStats { lower, upper });
+            let stats = BytesStats::Atomic(AtomicBytesStats {
+                lower,
+                upper,
+                kind: Some(AtomicBytesStatsKind::ProtoDatum),
```

Comment:
At first glance this looks like a [compatibility hazard](https://materializeinc.slack.com/archives/C03P1EGB7A7/p1719326449752639); thoughts? Anyways worth running nightlies probably...

Reply:
Eeek yeah you're right, let me update this!